test/models/differential_expression_parameters_test.rb

```diff
@@ -70,4 +71,9 @@ class DifferentialExpressionParametersTest < ActiveSupport::TestCase
     assert_equal '--annotation-name', DifferentialExpressionParameters.to_cli_opt(:annotation_name)
     assert_equal '--foo', DifferentialExpressionParameters.to_cli_opt('foo')
   end
+
+  test 'should provide default ram_in_mb for DE jobs' do
```

Comment:
Blocker: this trivial typo could mislead maintainers.

```suggestion
  test 'should provide default ram_in_gb for DE jobs' do
```

Reply:
Yes, exactly - it's totally fine to label something as a blocker, but proactively approving for more trivial changes makes the process smoother.  Thanks again!